internal/tools/looker/lookerrunlook/lookerrunlook.go

```diff
@@ -126,17 +126,30 @@ func (t Tool) Invoke(ctx context.Context, params tools.ParamValues, accessToken
 
 	look_id := paramsMap["look_id"].(string)
 	limit := int64(paramsMap["limit"].(int))
+	limitStr := fmt.Sprintf("%d", limit)
```

Comment:
nit: you may be able to simplify to  `limitStr := fmt.Sprintf("%d", paramsMap["limit"])`